src/main/java/pl/trainingapp/controller/requests/ExerciseRequest.java

```diff
@@ -0,0 +1,12 @@
+package pl.trainingapp.controller.requests;
+
+import lombok.Data;
+
+@Data
+public class ExerciseRequest {
+
+    private int id;
```

Comment:
You don't use id. Your mocked database aleady is autogenerating id and in the end database or hibernate will be responsible for it

Reply:
ok, deleted id